code/modules/mob/living/carbon/human/human.dm

```diff
@@ -66,6 +66,7 @@
 
 /mob/living/carbon/human/Destroy()
 	QDEL_LIST(bodyparts)
+	QDEL_LIST(splinted_limbs)

```

Comment:
This isn't how this works; you're deleting the limbs in the list, when they've already been deleted since they're also in `bodyparts`.